website/www/site/content/en/documentation/runners/spark.md

```diff
@@ -181,12 +181,20 @@ pip install apache_beam
 {{< /highlight >}}
 
 {{< paragraph class="language-py" >}}
-As of now you will need a copy of Apache Beam's source code. You can
-download it on the [Downloads page](/get-started/downloads/). In the future there will be pre-built Docker images
-available.
+Starting from Beam 2.20.0, pre-built Spark Job Service Docker images are available at 
+[Docker Hub](https://hub.docker.com/r/apache/beam_spark_job_server).
 {{< /paragraph >}}
 
-{{< paragraph class="language-py" >}}1. Start the JobService endpoint: `./gradlew :runners:spark:job-server:runShadow`{{< /paragraph >}}
+{{< paragraph class="language-py" >}}
+For older Beam versions, you will need a copy of Apache Beam's source code. You can
+download it on the [Downloads page](/get-started/downloads/).
+{{< /paragraph >}}
+
+{{< paragraph class="language-py" >}}
+1. Start the JobService endpoint:
+    * with Docker (default): `docker run --net=host apache/beam_spark_job_server:latest`
```

Comment:
It's not really a "default." I think "preferred" is more accurate.

Reply:
Agree, fixed